src/components/topic/TopicService.js

```diff
@@ -71,10 +71,15 @@ goog.require('ga_permalink');
         });
 
         // Returns a promise that is resolved when topics are loaded
-        this.getTopics = function() {
+        this.loadConfig = function() {
           return topicsP;
```

Comment:
yes but I thnik  we don't care because the  call of  'then' is like the first `gaTopicChange` event it miss.

Hard to explain:)